code/game/turfs/strata.dm

```diff
@@ -5,6 +5,13 @@
 /turf/open/gm/grass/weedable/is_weedable()
 	return FULLY_WEEDABLE
 
+//just in case
+
+/turf/open/gm/grass/grass1/weedable/ //inherit from general turfs
```

Comment:
```suggestion
/turf/open/gm/grass/grass1/weedable //inherit from general turfs
```

Reply:
this is probably not on only one instance.... will fix that....